File: docs/code/RunModel/Matlab_Example/sum_scalar.m
x = zeros(3,1);
x(1) = <var1>;
x(2) = <var11>;
x(3) = <var111>;
output = sum(x);
if ~ exist('OutputFiles', 'dir')
status = mkdir('OutputFiles');
end
csvwrite(sprintf('OutputFiles/oupt.out'),output);

File: tests/unit_tests/run_model/sum_scalar.py
import numpy as np
import os

x = np.zeros(3)
x[0] = <var1>
x[1] = <var11>
x[2] = <var111>
output = sum(x)

if not os.path.isdir('OutputFiles'):
    os.mkdir('OutputFiles')

np.save('OutputFiles/oupt.npy', output)
